dubbo.go

```diff
@@ -137,3 +145,63 @@ func (ins *Instance) NewServer(opts ...server.ServerOption) (*server.Server, err
 	}
 	return srv, nil
 }
+
+func (ins *Instance) Start() (err error) {
```

Comment:
There is no need to expose this method?

Reply:
Yeah, I don't think it needs to be exposed